src/modules/Makefile.in

```diff
@@ -25,7 +25,7 @@ MODULELD=@MODULELD@
 MODULECC=@MODULECC@
 MODULESHCFLAGS=@MODULESHCFLAGS@
 MODULECXX=@MODULECXX@
-MODULESHCXXFLAGS=@MODULESHCXXFLAGS@ -D__STDC_LIMIT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_CONSTANT_MACROS -std=c++17
+MODULESHCXXFLAGS=@MODULESHCXXFLAGS@ -Wno-mismatched-new-delete -Wno-sign-compare -D__STDC_LIMIT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_CONSTANT_MACROS -DPROTOBUF_USE_DLLS -std=c++20
```

Comment:
What code triggers these warnings, and if it's ours, can we not fix it so these suppressions are unnecessary?

Reply:
its in 3rd party code grpc/protobuf, have to live with it for now